ZIGSIMPlus/Models/AppSettingModel.swift

```diff
@@ -155,10 +165,15 @@ public class AppSettingModel {
         if command != .imageDetection {
             isCameraUsed = (isCameraUsed || isActiveByCommand[.imageDetection]!)
         }
-        
+
         return isCameraUsed
     }
 
+    public func isCameraEnabled() -> Bool {
```

Comment:
This was renamed in #39 .

Reply:
oh, I forgot to remove it on merge... fixed in 1c62afe :bow: